### ISSUE TITLE
Armor trims aren't visible on barbaric helmet

### ISSUE THREAD

**Issue by JustSkay** [MehVahdJukaar/ohmygoat#21]
Not sure if that's a bug, but since it is possible to apply trims on them i would assume that it is. If it's not, there's really no reason to not be able to apply trims to it

**Comment by MehVahdJukaar** [comment]
Yeah you should probably not be able to apply them. Tho you can apply them on turtle one so I might change ut

**Comment by JustSkay** [comment]
yeah it would actually be nice if you could apply them and see them
